Additional errors for 42NAC, 50N26, 51N7A (#398)
neo4j/docs-status-codes: modules/ROOT/pages/errors/gql-errors/51N7A.adoc

= 51N7A

== Status description

> error: system configuration or operation exception - no admin user candidate. No admin user candidate could be found. Please use `neo4j-admin dbms set-default-admin` to select a valid admin.

Reviewer: ```suggestion
error: system configuration or operation exception - no admin user candidate. No admin user candidate is found. Use `neo4j-admin dbms set-default-admin` to select a valid admin.
```
